model_analyzer/config/generate/quick_run_config_generator.py

```diff
@@ -424,17 +426,24 @@ def _get_next_model_config(
             self._coordinate_to_measure, dimension_index
         )
 
+        model_config_params = deepcopy(model.model_config_parameters())
+        if model_config_params:
+            model_config_params.pop("max_batch_size", None)
+
+            # This is guaranteed to only generate one combination (check is in config_command)
+            param_combo = GeneratorUtils.generate_combinations(model_config_params)[0]
```

Comment:
can we recheck here for sanity? As an assert? I don't like this coupling, having one section of the code make an unchecked assumption about other code.
param_combos = blah
if (len != 1) raise error
param_combo = param_combos[0]

Reply:
Assertion added